scripts/cpp-ethereum/eth_trusty.docker

```diff
@@ -1,7 +1,13 @@
 FROM ubuntu:trusty
 
-RUN apt update
-RUN apt -y install libleveldb-dev cmake g++ git
+RUN apt-get update
+RUN apt-get -y install software-properties-common python-software-properties
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
+RUN apt-get -y install gcc libleveldb-dev git curl make gcc-7 g++-7
+RUN ln -sf /usr/bin/gcc-7 /usr/bin/gcc
```

Comment:
Since above `gcc` is installed, will that not conflict with `/usr/bin/gcc`, i.e. doesn't that needs to be deleted first?

Reply:
`-f` deals with that. I think we need to install gcc first so that it installs some other required packages, but I'm not sure.

Reply:
Ah ok.